siliconcompiler/pdks/skywater130.py

```diff
@@ -112,6 +112,9 @@ def setup(chip):
     chip.set('pdk', process, 'var', 'openroad', 'pin_layer_vertical', stackup, 'met2')
     chip.set('pdk', process, 'var', 'openroad', 'pin_layer_horizontal', stackup, 'met3')
 
+    # Hide the 81/4 'areaid.standardc' layer by default; it puts opaque purple over most core areas.
```

Comment:
can you use the `areaid.standardc` in the list instead? I think that would be clearer than the GDS layer/purpose

Reply:
We could accept either; it looks like `layer.name` returns a value in the format `"areaid.standardc - 81/4"`

Reply:
I think the layer name is clearer

Reply:
I put the layer name into the Sky130 PDK, but made the KLayout script accept either, in case someone wants to use a PDK without a clear layer properties file, or one that is being actively developed, or something like that.